code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1235,3 +1235,82 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "glass_yellow"
 	glass_name = "Eggnog"
 	glass_desc = "For enjoying the most wonderful time of the year."
+
+// IPC ALCHOHOL WOOOOOOOOOOO
+/datum/reagent/consumable/ethanol/synthanol
+	name = "Synthanol"
+	id = "synthanol"
+	description = "A runny liquid with conductive capacities. Its effects on synthetics are similar to those of alcohol on organics."
+	reagent_state = LIQUID
+	color = "#1BB1FF"
+	process_flags = ORGANIC | SYNTHETIC
+	boozepwr = 20
+	glass_icon_state = "synthanolglass"
+	glass_name = "Glass of Synthanol"
+	glass_desc = "The equivalent of alcohol for synthetic crewmembers. They'd find it awful if they had tastebuds too."
+
+/datum/reagent/consumable/ethanol/synthanol/robottears
+	name = "Robot Tears"
+	id = "robottears"
+	description = "An oily substance that an IPC could technically consider a 'drink'."
+	reagent_state = LIQUID
+	color = "#363636"
+	boozepwr = 50
```

Comment:
Should be lower, considering it's mixed down from synthanol.

Reply:
Changes